dag_in_context/src/schedule.rs

```diff
@@ -91,6 +91,22 @@ pub fn mk_sequential_schedule() -> Vec<String> {
       passthrough
       state-edge-passthrough))"
     )];
+    res.push(format!(
+        "
+(run-schedule
+  (repeat 2
+      {helpers}
+      swap-if)
+  {helpers}
+  rec-to-loop
+  {helpers})"
+    ));
+    res.push(format!(
+        "
+;; HACK: when INLINE appears in this string
+;; we perform inlining in this pass
```

Comment:
To make this comment clear, "when INLINE appears in this string, we perform inlining in this pass" => "when the keyword INLINE appears in the schedule, `optimize` in `lib.rs` inlines the input program before running this schedule"

Alternatively, we can have this function returns a `Schedule` struct, which makes this no longer a hack.

Reply:
Good idea- will follow up with another PR